lintro/utils/tsconfig.py

```
@@ -0,0 +1,579 @@
+def _resolve_extends_path(extends: str, base_dir: Path) -> Path | None:
+    """Resolve an ``extends`` value to an absolute path.
+
+    Args:
+        extends: The extends value (relative path or package name).
+        base_dir: Directory of the config file containing the extends.
+
+    Returns:
+        Resolved path or ``None`` if unresolvable.
+    """
+    # Relative paths
+    if extends.startswith(".") or extends.startswith("/"):
```

Ruff (0.15.9)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)